README.md

```diff
@@ -37,15 +39,26 @@
 
 ### API Documentation
 
-- The updated API documentation is created with [Postman](https://www.postman.com/) and can be found [here](https://documenter.getpostman.com/view/24955418/2sA2xmUqFr).
+- The autogenerated OpenAPI/Swagger documentation is found at server `/api` endpoint.
+  - Fully intractable testing interface
+  - Includes full request/response schemas and example requests/responses
+  - Currently deployed documentation is found here: https://tablebook.onrender.com/api
+  - The OpenAPI documentation is created from [Zod](https://zod.dev/) endpoint schemas. Using [zod-to-openapi](https://www.npmjs.com/package/@asteasolutions/zod-to-openapi).
+  - The OpenAPI intractable documentation endpoint is created with [swagger-ui-express](https://www.npmjs.com/package/swagger-ui-express)
```

Comment:
``` Suggestion
  - The OpenAPI interactable documentation endpoint is created with [swagger-ui-express](https://www.npmjs.com/package/swagger-ui-express)
```
Should this say interactable? Intractable seems to mean "hard to control or deal with".